forge-gui/res/cardsfolder/upcoming/jo_grant.txt

```diff
@@ -0,0 +1,11 @@
+Name:Jo Grant
+ManaCost:2 W
+Types:Legendary Creature Human Soldier
+PT:3/2
+S:Mode$ Continuous | Affected$ Card.Historic | EffectZone$ Battlefield | AffectedZone$ Hand | AddKeyword$ Cycling:2 W | Description$ Each historic card in your hand has cycling {2}{W}. ({2}{W}, Discard that card: Draw a card.)
```

Comment:
Needs to affect only the historic cards in your hand, not historic cards in all players' hands.

Reply:
Good catch thanks